ui/src/plugins/Notify.js

```diff
@@ -64,6 +64,14 @@ function logError (error, config) {
 const Notifications = {
   name: 'QNotifications',
 
+  // hide App from Vue devtools
+  devtools: { hide: true },
```

Comment:
Isn't this a Vue 3 feature of devtools?
Why is this needed?

Reply:
Vue devtools beta works with vue2, and it:
- shows the top level app
- borks on getting router for these plugins